x-pack/plugins/rollup/public/crud_app/store/actions/delete_jobs.js

```diff
@@ -30,13 +35,21 @@ export const deleteJobs = (jobIds) => async (dispatch, getState) => {
       type: UPDATE_JOB_FAILURE,
     });
 
-    return toastNotifications.addDanger(error.data.message);
+    return showApiError(error, i18n.translate('xpack.rollupJobs.deleteAction.fatalErrorTitle', {
+      defaultMessage: 'Rollup Job Wizard delete jobs',
+    }));
   }
 
   if (jobIds.length === 1) {
-    toastNotifications.addSuccess(`Rollup job '${jobIds[0]}' was deleted`);
+    toastNotifications.addSuccess(i18n.translate('xpack.rollupJobs.deleteAction.successSingleNotificationTitle', {
+      defaultMessage: `Rollup job '{jobId}' was deleted`,
+      values: { jobId: jobIds[0] },
+    }));
   } else {
-    toastNotifications.addSuccess(`${jobIds.length} rollup jobs were deleted`);
+    toastNotifications.addSuccess(i18n.translate('xpack.rollupJobs.deleteAction.successMultipleNotificationTitle', {
+      defaultMessage: '{count} rollup jobs were deleted',
```

Comment:
Do we want to handle pluralization?

Reply:
Not necessary, since this branch of the logic only handles multiple jobs.